### PR TITLE
Add description_content_type field to Release

### PR DESCRIPTION
**Pull request by di**
This PR adds the `description_content_type` field to the `Release` model and captures it on upload.

### CODE REVIEW

**Comment by hallazzang** [comment]
I'm waiting for this PR to be merged

**Comment by di** [comment]
@hallazzang Thanks for your interest. You might want to subscribe to https://github.com/pypa/warehouse/issues/869 where I've been tracking most of the work necessary (including this PR), specifically in [this comment](https://github.com/pypa/warehouse/issues/869#issuecomment-340928703).

File: warehouse/forklift/legacy.py [hunk 1]
```diff
@@ -16,6 +16,7 @@
 import re
 import tempfile
 import zipfile
+from cgi import parse_header
 
 from cgi import FieldStorage
```
Comment (merwok) [inline warehouse/forklift/legacy.py]: Seems weird to use old cgi module functions where Pyramid provides the needed parsers.

Comment (di) [reply warehouse/forklift/legacy.py]: @merwok Thanks for the review. Assuming you're talking about `parse_header`, would you mind pointing me to the equivalent function which Pyramid provides?

**Comment by di** [comment]
For those watching, this PR does not get you descriptions in Markdown, [there are still a few outstanding blockers here](https://github.com/pypa/warehouse/issues/869#issuecomment-340928703) (but we're close).

File: warehouse/forklift/legacy.py (hunk 1 above)
Comment (merwok) [reply warehouse/forklift/legacy.py]: Sorry for missing this!  I’m talking about both parse_header and FieldStorage; Pyramid parses these and surely exposes a way to get Python objects for them, but I can’t look for it right now.

**Pull request merged**